spec/spec_helper.rb

```diff
@@ -32,7 +32,7 @@
 
 ALL_DTYPES = [:byte,:int8,:int16,:int32,:int64, :float32,:float64, :object,
   :complex64, :complex128]
-
+  
```

Comment:
I have removed the INTEGER_DTYPES array but I think it can be kept here for future references. Also, if you suggest ALL_STYPES array can also be made for the specs.